components/src/core/components/Input/Time/TimeInput.vue

```diff
@@ -13,9 +13,13 @@
         tabindex="0"
```

Comment:
variable names feel bit verbose, this is _subjective feedback_ but IMO we can refactor fields internal variable names such as  `timeFormatToDisplay` to simply `displayFormat` since `time` is inferred inside timepicker.